packages/webapp/pages/squads/[handle]/index.tsx

```diff
@@ -176,20 +209,73 @@ SquadPage.layoutProps = mainFeedLayoutProps;
 
 export default SquadPage;
 
-export async function getStaticPaths(): Promise<GetStaticPathsResult> {
-  return { paths: [], fallback: true };
-}
-
 interface SquadPageParams extends ParsedUrlQuery {
   handle: string;
 }
 
-export function getStaticProps({
+export async function getServerSideProps({
   params,
-}: GetStaticPropsContext<SquadPageParams>): GetStaticPropsResult<SourcePageProps> {
-  return {
-    props: {
-      handle: params.handle,
-    },
+  query,
+  res,
+}: GetServerSidePropsContext<SquadPageParams>): Promise<
+  GetServerSidePropsResult<SourcePageProps>
+> {
+  const { handle } = params;
+  const { userid: userId, cid: campaign } = query;
+
+  const setCacheHeader = () => {
+    res.setHeader(
+      'Cache-Control',
+      `public, max-age=0, must-revalidate, s-maxage=${oneHour}`,
```

Comment:
I would even put this to one day, since squad metadata does not change as often.

Reply:
Makes sense. Longer but reasonable the better. Is there a way though to manually invalidate the cache?

Reply:
I personally think one hour is fine, in case we do decide to make drastic changes it can be easier.
An hour is also pretty good considering what the page does.

Reply:
Ok, let's leave it.

> Is there a way though to manually invalidate the cache?

Same as for static, redeploy or clearing through data cache interface https://vercel.com/dailydotdev/daily-webapp/settings/data-cache